src/devtools/client/inspector/computed/components/ComputedApp.tsx

```diff
@@ -7,7 +7,9 @@ export default function ComputedApp() {
   return (
     <div id="sidebar-panel-computedview" className="theme-sidebar inspector-tabpanel">
       <ComputedToolbar />
+      <div style={{overflow:"scroll",height:"220px"}}>
```

Comment:
Hey @Roaster05, I feel it's not a good practice to give inline css, rather we can change the css in `computed` css file.

Reply:
Hey Manav thanks for your feedback but the previous styling was also inline therefore I preferred not to change that if the maintainers want the css to be mentioned explicitly in the styling file i will do the same